client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -377,19 +384,60 @@ class CheckoutThankYouHeader extends PureComponent {
 					<div className="checkout-thank-you__header-copy">
 						<h1 className="checkout-thank-you__header-heading">{ this.getHeading() }</h1>
 
-						<h2 className="checkout-thank-you__header-text">{ this.getText() }</h2>
+						{ primaryPurchase && isPlan( primaryPurchase ) && isSimplified ? (
+							this.renderSimplifiedContent()
+						) : (
+							<h2 className="checkout-thank-you__header-text">{ this.getText() }</h2>
+						) }
 
 						{ this.getButton() }
 					</div>
 				</div>
 			</div>
 		);
 	}
+
+	renderSimplifiedContent() {
+		const { translate, primaryPurchase } = this.props;
+		const messages = [
+			translate(
+				'Your site is now on the {{strong}}%(productName)s{{/strong}} plan. ' +
+					'Enjoy your powerful new features!',
+				{
+					args: { productName: primaryPurchase.productName },
+					components: { strong: <strong /> },
+				}
+			),
+		];
+		if ( this.props.siteUnlaunchedBeforeUpgrade ) {
+			messages.push(
+				translate(
+					"Your site has been launched. You can share it with the world whenever you're ready."
+				)
+			);
+		}
+
+		if ( messages.length === 1 ) {
+			return <h2 className="checkout-thank-you__header-text">{ messages[ 0 ] }</h2>;
+		}
+
+		const CHECKMARK_SIZE = 24;
```

Comment:
Should SCREAMING_SNAKE_CASE naming convention be used rather for application constants outside a function scope?